src/System.Linq.Expressions/src/System/Runtime/CompilerServices/ReadOnlyCollectionBuilder.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
+using System.Diagnostics.Contracts;
```

Comment:
Contract.Ensures isn't useful, as we don't use the contract rewriter and thus don't set the flags that enable this method. It's a nop.